docs/transactions.md

```diff
@@ -1245,7 +1245,7 @@ If a Revocation Distribution Point (such as RFC5280 Certificate Revocation List)
 
 **Status: Implemented**
 
```

Comment:
I would add a new section GET_X509_CERTS_BY_SUBJECT_KEY_ID instead of modifying existing one to make it clear that we support querying by SUBJECT_KEY_ID.
- CLI command: `dcld query pki x509-cert --subject-key-id=<hex string>`
- REST:  `/dcl/pki/certificates?subjectKeyId={subjectKeyId}`